python_modules/dagster/dagster_tests/scheduler_tests/test_scheduler_run.py

```diff
@@ -719,152 +720,33 @@ def test_error_load_code_location(instance: DagsterInstance, executor: ThreadPoo
     ) as workspace_context:
         fake_origin = _get_unloadable_schedule_origin()
         freeze_datetime = feb_27_2019_one_second_to_midnight()
-        with pendulum_freeze_time(freeze_datetime):
+        with freeze_time(freeze_datetime):
             schedule_state = InstigatorState(
                 fake_origin,
                 InstigatorType.SCHEDULE,
                 InstigatorStatus.RUNNING,
-                ScheduleInstigatorData("0 0 * * *", pendulum.now("UTC").timestamp()),
+                ScheduleInstigatorData("0 0 * * *", get_current_timestamp()),
             )
             instance.add_instigator_state(schedule_state)
 
-        freeze_datetime = freeze_datetime.add(seconds=1)
-        with pendulum_freeze_time(freeze_datetime):
-            evaluate_schedules(workspace_context, executor, pendulum.now("UTC"))
+        freeze_datetime = freeze_datetime + relativedelta(seconds=1)
+        with freeze_time(freeze_datetime):
+            evaluate_schedules(workspace_context, executor, get_current_datetime_in_utc())
 
             assert instance.get_runs_count() == 0
 
             ticks = instance.get_ticks(fake_origin.get_id(), schedule_state.selector_id)
 
             assert len(ticks) == 0
 
-        freeze_datetime = freeze_datetime.add(days=1)
-        with pendulum_freeze_time(freeze_datetime):
-            evaluate_schedules(workspace_context, executor, pendulum.now("UTC"))
+        freeze_datetime = freeze_datetime + relativedelta(days=1)
+        with freeze_time(freeze_datetime):
+            evaluate_schedules(workspace_context, executor, get_current_datetime_in_utc())
             assert instance.get_runs_count() == 0
             ticks = instance.get_ticks(fake_origin.get_id(), schedule_state.selector_id)
             assert len(ticks) == 0
 
 
-@pytest.mark.parametrize("executor", get_schedule_executors())
-def test_grpc_server_down(instance: DagsterInstance, executor: ThreadPoolExecutor):
```

Comment:
this test was already reliably hanging locally in master and did some worth the complexity to maintain